packages/rancher-prom-to-teams/generated-changes/patch/templates/configmap.yaml.patch

```diff
@@ -0,0 +1,133 @@
+--- charts-original/templates/configmap.yaml
++++ charts/templates/configmap.yaml
+@@ -8,13 +8,13 @@
+ {{- if and (not .Values.prom2teams.connector) (not .Values.prom2teams.connectors) -}}
+ {{- fail "Invalid configuration: At least one connector must be provided"}}
+ {{- end -}}
```

Comment:
Drop this, since we are always adding one in on install and expect it to be changed outside of a Helm context